package.json

```diff
@@ -792,6 +797,16 @@
         "when": "editorLangId == sas && !SAS.hideRunMenuItem",
         "command": "SAS.runSelected",
         "key": "f3"
+      },
+      {
+        "key": "ctrl+/",
```

Comment:
I would suggest to make a single command, ctrl+/ to work as a toggle. It will match VS Code built-in behavior. I understand you're following the behavior in traditional SAS editor. But we're in VS Code. A user may also work on other languages and is used to Ctrl+/ to toggle comment. Actually in SAS Studio, Ctrl+/ works as a toggle too.